src/obs_inv_utils/search_engine.py

```diff
@@ -215,6 +273,31 @@ def process_inspect_tarball_resp(contents):
     tbl_factory.insert_obs_inv_items(tarball_files_meta)
 
 
+def default_datetime_converter(obj):
+   if isinstance(obj, datetime):
+      return obj.__str__()
+
+def post_aws_s3_cmd_result(raw_response, obs_cycle_time):
```

Comment:
same here. should this be inside of aws_s3_interface

Reply:
If you prefer, I can include the changes necessary to separate out the interface specificity in this PR, but it does make the PR bigger.

Reply:
same here. follow up PR will be better

Reply:
I'll create a follow-up PR for this one.